bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Wh_M125_ToAA_Tobbgg_LO_MLM/Wh_M125_ToAA__AMASS__Tobbgg_proc_card.dat

```diff
@@ -0,0 +1,18 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set gauge unitary
+set complex_mass_scheme False
+
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l = e+ mu+ ta+ e- mu- ta-
+define neut = ve vm vt ve~ vm~ vt~ 
+define V = w+ w-
+import model --modelname HAHM_MG5model_v3/HAHM_variableMW_v3_UFO
```

Comment:
The model name should be:
`import model --modelname HAHM_variableMW_v3_UFO`

Reply:
@Saptaparna we have already used that model.. so what's the issue I didn't get, sorry.. can you please clarify?

Reply:
This syntax:

`import model --modelname HAHM_variableMW_v3_UFO`

works for me, while this syntax `import model --modelname HAHM_MG5model_v3/HAHM_variableMW_v3_UFO` does not.